developer-docs/upgrade/build_n_deploy.md

```diff
@@ -7,11 +7,36 @@ allowSearch: true
 keywords: Upgrade, Sunbird 2.3.0, Core, elasticsearch
 ---
 
-## Overview
+**New load balancer setup for Swarm manager**
+
+(Optional - Refer the variable sunbird_swarm_manager_lb_ip in inventory for comments)
```

Comment:
@keshavprasadms need clarity on this statement

Reply:
Added more verbosity.